frontend/src/scenes/groups/groupsListLogic.ts

```diff
@@ -1,20 +1,26 @@
 import { kea } from 'kea'
 import api from 'lib/api'
 import { groupsAccessLogic } from 'lib/introductions/groupsAccessLogic'
-import { capitalizeFirstLetter } from 'lib/utils'
 import { teamLogic } from 'scenes/teamLogic'
 import { urls } from 'scenes/urls'
 import { Noun, groupsModel } from '~/models/groupsModel'
 import { Breadcrumb, Group } from '~/types'
 import type { groupsListLogicType } from './groupsListLogicType'
+import { capitalizeFirstLetter } from 'lib/utils'
 
 export interface GroupsPaginatedResponse {
     next: string | null
     previous: string | null
     results: Group[]
 }
 
+export interface GroupsListLogicProps {
+    groupTypeIndex: number
+}
+
 export const groupsListLogic = kea<groupsListLogicType>({
+    props: {} as GroupsListLogicProps,
+    key: (props: GroupsListLogicProps) => props.groupTypeIndex,
```

Comment:
Previously there could only be one instance of `groupsListLogic` globally, so we couldn't load multiple group types at once. Now that this is keyed, we can